projects/packages/connection/src/class-manager.php

```diff
@@ -484,12 +484,15 @@ private function internal_verify_xml_rpc_signature() {
 	/**
 	 * Returns true if the current site is connected to WordPress.com and has the minimum requirements to enable Jetpack UI.
 	 *
+	 * This method is deprecated since Jetpack 9.6.0. Please use one has_connected_owner instead.
+	 *
+	 * Since this method has a wide spread use, we decided not to throw any deprecation warnings for now.
+	 *
+	 * @deprecated 9.6.0
+	 * @see Manager::has_connected_owner
 	 * @return Boolean is the site connected?
 	 */
 	public function is_active() {
-		if ( ( new Status() )->is_no_user_testing_mode() ) {
-			return $this->is_connected();
-		}
 		return (bool) $this->get_tokens()->get_access_token( true );
```

Comment:
In the old `is_connected` method, we check for both a token and a blog ID. Should we keep doing the same here?

Reply:
Hi Jeremy! Thanks for reviewing!

We are not changing any functionality here, just removing the user-less mode filter (which was never used in production anyway).
`is_connected` still checks for both a blog token and a blog ID.
`is_active` (to be deprecated) still checks for the connection owner token as always.